src/backend/joanie/core/templatetags/extra_tags.py

```diff
@@ -5,6 +5,7 @@
 from django.utils.translation import gettext as _
 
 from joanie.core.utils import image_to_base64
+from joanie.core.utils import iso8601_to_duration as iso8601_to_duration_utility
```

Comment:
Not quit sure this is currently a good idea to create an util for that.

Here are my points : 
- You are currently using this logic at a single point, so I don't think this is a good idea to abstract this logic at first
- There are currently a weakness in your tests as you are testing the utils but not this template tags. I agree that right now, tests will be exactly the same but if later we update the logic of this template tag, no test will be able to warn us about a regression.